src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -54,3 +54,6 @@ dataverse.pid.datacite.rest-api-url=https://api.test.datacite.org
 
 # Handle.Net
 dataverse.pid.handlenet.index=300
+
+# Directory to load external exporters from
+dataverse.exporters.directory=/tmp/exporters
```

Comment:
Not sure I like this. How about `${com.sun.aas.instanceRoot}/plugins/exporters`?

Also, see my remark on `ExportService` above about how this might log strange warning when absolutely not necessary.